components/hab/src/command/pkg/download.rs

```diff
@@ -234,8 +264,9 @@ impl<'a> DownloadTask<'a> {
 
     fn determine_latest_from_ident<T>(&self,
                                       ui: &mut T,
-                                      ident: &PackageIdent,
-                                      target: PackageTarget)
+                                      _channel: &ChannelIdent,
```

Comment:
Minor - underscore is a pattern for an unused variable, so would be good to change to use a non-underscore channel var

Reply:
Thanks, renamed variable.